fetch.bs

```diff
@@ -2689,6 +2689,9 @@ functionality.
 <p>A <a for=fetch>fetch group</a> holds an ordered list of
 <dfn lt="fetch record" export for="fetch group" id=concept-fetch-record>fetch records</dfn>.
 
+<p>A <a for=fetch>fetch group</a> holds an ordered list of
```

Comment:
> Each [environment settings object ](https://html.spec.whatwg.org/multipage/webappapis.html#environment-settings-object)has an associated fetch group .
> <p>A <a for=fetch>fetch group</a> holds an ordered list of <dfn export for="fetch group" id=concept-defer=fetch-record>deferred fetch records</dfn>.

The above implies that a deferred fetch records belong to a Document, even after the Document is unloaded. In practice, user agent will have to put deferred fetch records in other context outside Document (e.g. browser process) in implementation. Does it make sense?

Reply:
Exactly. This "association" can be cross-process as far as the spec is concerned. That's also why fetch group and document are not *exactly* the same thing, they have a 1:1 relationship but `fetch-group` outlives `document` in order to send deferred/keepalive fetches

Reply:
Ok. I was confused as `environment settings object` mentions it has a `A realm execution context` which is clearly bound to a renderer (JS context)'s lifetime.